### PR TITLE
Update README.md to personalize to our app

### LINKED ISSUE #12: Update README UI mockup
Add UI mockup of intended final product to README.

### PR DESCRIPTION
**Pull request by craigtonlian**
README.md does not conform to standards set out in Week 7 scope of tP.

- Updating the CI badge link to our project's CI will reflect our project's state. 
- Changing the app UI image to Staff-Snap's preliminary design will better reflect our design progress. 
- Additional links will improve the navigation for readers. 
- Minor UI tweaks to craigtonlian.md for personal taste. 
- Updating MarkBind's site.json and default.md removes remnants of AB3.

Let's implement these changes.

Closes #12.

### CODE REVIEW

**Review by CelestineTan03 — COMMENTED** [review]
Lgtm! Just some nits to fix 😊

File: README.md [hunk 1]
```diff
@@ -1,32 +1,40 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+# Staff-Snap
+
+[![CI Status](https://github.com/craigtonlian/tp/actions/workflows/gradle.yml/badge.svg)](https://github.com/craigtonlian/tp/actions)
```
[the remark this answers is not in the document]
Comment (craigtonlian) [reply README.md:3]: Good catch! 😊

File: README.md [hunk 2]
```diff
@@ -1,32 +1,40 @@
...
-**Payroll**
-* Update on a monthly basis whether an employee has been paid for that payment period
-* Update on a monthly basis if an entire department has been paid for that payment period
-* Generate a preliminary bonus allocation based on the performance analytics score as a basis for splitting the bonus amount among all eligible staff, depending on the sum available for disbursal
+## Issues and Bugs
+Spot a bug? Let us know by launching a Bug Report at the [Issues tab](https://github.com/AY2324S1-CS2103T-W08-1/tp/issues)!
 
+## Links
+- [User Guide](https://ay2324s1-cs2103t-w08-1.github.io/tp/UserGuide.html)
+- [User Stories](https://ay2324s1-cs2103t-w08-1.github.io/tp/DeveloperGuide.html)
```
[the remark this answers is not in the document]
Comment (craigtonlian) [reply README.md:38]: I like the idea of directing to Developer Guide 👍

File: docs/images/Ui.png
[the remark this answers is not in the document]
Comment (craigtonlian) [reply docs/images/Ui.png:1]: I agree 😄

**Review by CelestineTan03 — APPROVED** [review]
Lgtm! 😊

**Pull request merged**